cri-o-centos/Dockerfile

```diff
@@ -14,7 +14,8 @@ LABEL com.redhat.component="cri-o" \
 COPY tmpfiles.template config.json.template service.template /exports/
 
 RUN yum-config-manager --nogpgcheck --add-repo https://cbs.centos.org/repos/virt7-docker-common-candidate/x86_64/os/ && \
-    yum install --nogpgcheck --setopt=tsflags=nodocs -y cri-o cri-o-cni && \
+    yum install --nogpgcheck --setopt=tsflags=nodocs -y iptables cri-o && \
```

Comment:
it is called `containernetworking-cni-` on CentOS:

https://cbs.centos.org/repos/virt7-docker-common-candidate/x86_64/os/Packages/

It didn't fail before as we had no `rpm -V` and yum install is happy if any package is installed.  It is not needed anyway as it is a dependency of `cri-o`